scripts/xe-backup-metadata

```diff
@@ -217,6 +221,15 @@ if [ ${leave_mounted} -eq 0 ]; then
      echo ${metadata_version} >> ${mnt}/.ctxs-metadata-backup
   fi
 
+  # check the usage of the backup VDI
+  usage=`df -H "$PWD" | sed -n "2p" | awk '{ print $5 }' | cut -d'%' -f1`
```

Comment:
`$PWD` is equivalent to `.`; flag `-H` is not required because it does not affect the capacity field; `cut -d'%' -f1` could be also implemented as `tr -d '%'`.

Reply:
Thanks. Updated.
The log for one success backup is:
```
[root@xenserver-lrwebsln ~]# xe-backup-metadata -v
Using SR: Local storage
Using existing backup VDI: ede3d0da-1d9f-4254-bf20-a51826e6e5c9
Creating VBD: ffa081d6-bb72-fbaf-dbc8-b44199c92c1d
Plugging VBD: /dev/sm/backend/1a027555-3c78-1493-a629-bd125e7501a7/ede3d0da-1d9f-4254-bf20-a51826e6e5c9
Mounting filesystem: /var/run/pool-backup-ede3d0da-1d9f-4254-bf20-a51826e6e5c9
Checking backup VDI space usage: 4%
Rotating old backups: found 16
Backing up pool database: done
Cleaning old VM metadata: done
Backing up SR metadata: done
Backing up VM metadata: 
done
Backing up Template metadata: 
done
Unplugging VBD: . done
```

Reply:
Failed one(set up alert to 3% for testing):
```
[root@xenserver-lrwebsln ~]# xe-backup-metadata -v
Using SR: Local storage
Using existing backup VDI: ede3d0da-1d9f-4254-bf20-a51826e6e5c9
Creating VBD: 84fe473c-5e08-0d5d-424b-0907787d3a7b
Plugging VBD: /dev/sm/backend/1a027555-3c78-1493-a629-bd125e7501a7/ede3d0da-1d9f-4254-bf20-a51826e6e5c9
Mounting filesystem: /var/run/pool-backup-ede3d0da-1d9f-4254-bf20-a51826e6e5c9
Checking backup VDI space usage: 4%
Running out of space, you can use -d option to attach VDI and free more space, exit now.
Unplugging VBD: . done
```